fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/CascadesPlanner.java

```diff
@@ -279,6 +302,35 @@ public void setIndexScanPreference(@Nonnull IndexScanPreference indexScanPrefere
                 .build();
     }
 
+    public void setMaxTaskQueueSize(final int maxTaskQueueSize) {
+        configuration = this.configuration.asBuilder()
+                .setMaxTaskQueueSize(maxTaskQueueSize)
+                .build();
+    }
+
+    public void setMaxTotalTaskCount(final int maxTotalTaskCount) {
+        configuration = this.configuration.asBuilder()
+                .setMaxTotalTaskCount(maxTotalTaskCount)
+                .build();
+    }
+
+    @Nonnull
+    public RecordQueryPlannerConfiguration getConfiguration() {
+        return configuration;
+    }
+
+    public void setConfiguration(@Nonnull final RecordQueryPlannerConfiguration configuration) {
+        this.configuration = configuration;
+    }
+
+    private boolean taskQueueSizeExceeded(final RecordQueryPlannerConfiguration configuration, final int queueSize) {
```

Comment:
can you change name to `is.....` (code style).

Reply:
Done.